### PR TITLE
Combine Deployment and Deployment CLI pages

### PR DESCRIPTION
**Pull request by mkaput**
**Stack**:
- #1119
- #1118 ⬅
- #1117
- #1116

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: website/docs/tutorials/08-deploying/README.md [hunk 1]
```diff
@@ -1,17 +1,147 @@
...
+
+## Sending Invoke Transaction
+
+Protostar supports invoking contract functions from the CLI level.
+Invoking can be used to
+invoke [deployer contracts](https://community.starknet.io/t/universal-deployer-contract-proposal/1864)
+.
+To send an invoke transaction use [`protostar invoke`](/docs/cli-reference#invoke). The transaction
+can be [signed similarly to the declare transaction](#signing-a-declaration).
+
```
Comment (Arcticae) [inline website/docs/tutorials/08-deploying/README.md:94]: This should not be in `deploying`

File: website/docs/tutorials/08-deploying/README.md [hunk 2]
```diff
@@ -1,17 +1,147 @@
...
+# https://github.com/Shard-Labs/starknet-devnet
+["profile.devnet.protostar.deploy"]
+gateway-url="http://127.0.0.1:5050/"
+
+["profile.testnet.protostar.deploy"]
+network="testnet"
+
+["profile.mainnet.protostar.deploy"]
+network="mainnet"
```
Comment (Arcticae) [inline website/docs/tutorials/08-deploying/README.md:140]: This also does not belong only in `deploying`

**Review by Arcticae — CHANGES_REQUESTED** [review]
I've presented an alternative division in #1121 which i think is better, but i might be biased because i've done that one :P

**Comment by mkaput** [comment]
@Arcticae I like your work more than this. I'll close this PR

**Pull request closed without merging**